sentiment.py

```diff
@@ -0,0 +1,27 @@
+import string
+
+negative_words = ["bad", "ugly", "terrible", "awful", "stupid", "mad", "angry", "sad"]
+counter = 0
+
+def sentiment_analysis(sentence: str) -> str:
+    global counter
+    split_sentence = sentence.lower().translate(str.maketrans('', '', string.punctuation)).split(" ")
+
+    for neg_word in negative_words:
+        for word in split_sentence:
```

Comment:
code will be slightly efficient if you were checking for the existing of the negative word in the split_sentence before 
the second loop